### PR TITLE
improve booster release scripts

### PR DESCRIPTION
**Pull request by Ladicek**

### CODE REVIEW

File: generate_licenses.sh [hunk 1]
```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
```
Comment (michalszynkiewicz) [inline generate_licenses.sh:1]: why, oh why can't it figure out this is the same file? :(

**Review by michalszynkiewicz — APPROVED** [review]

**Comment by michalszynkiewicz** [comment]
I don't know this code but your changes look good to me.
We have 53 less lines of code and it's all a bit simpler :)

File: generate_licenses.sh (hunk 1 above)
Comment (Ladicek) [reply generate_licenses.sh:1]: Too much changes I'm afraid. I was too drastic :-)

**Comment by Ladicek** [comment]
Thanks @michalszynkiewicz! Yea I figured we can easily distinguish community vs. product based on directory name, which makes the code shorter and usage simpler.

**Pull request merged**